frontend/src/components/plugins/flows/utils.js

```diff
@@ -58,9 +59,19 @@ function addNode(
     });
   } else if (nodeToAdd.playbooks_choice) {
     nodeToAdd.playbooks_choice.forEach((child) => {
-      const playbookConfig = playbooksStored.find(
-        (plugin) => plugin.name === child,
-      );
+      let playbookConfig = {};
+      playbookConfig = playbooksStored.find((plugin) => plugin.name === child);
+      if (playbookConfig === undefined) {
+        playbookConfig = {};
+        playbookConfig.id = `${child}`;
+        playbookConfig.name = child;
+        playbookConfig.description =
+          "The playbook is not enabled or configured for this user.";
```

Comment:
I'll make this description more explicit. What is the consequence of this ? The pivot will never run ? pivot will run, but always fail ? I suggest to make more explicit to the user.